internal/driver/deviceclient.go

```diff
@@ -105,126 +105,186 @@ func calculateAddressLength(primaryTable string, valueType sdkModel.ValueType) u
 	return length
 }
 
-func TransformDataBytesToResult(req *sdkModel.CommandRequest, dataBytes []byte, commandInfo *CommandInfo) (*sdkModel.CommandValue, error) {
-	var result = &sdkModel.CommandValue{}
+func TransformDataBytesToResult(req *models.CommandRequest, dataBytes []byte, commandInfo *CommandInfo) (*models.CommandValue, error) {
```

Comment:
Func shoud have godoc style comment

Reply:
@hahattan Please help to add the following comment:
TransformDataBytesToResult is used to transform the device's binary data to the specified value type as the actual result.